klippy/extras/sht3x.py

```diff
@@ -105,7 +105,20 @@ def _init_sht3x(self):
     def _sample_sht3x(self, eventtime):
         try:
             # Read measurment
-            params = self.i2c.i2c_read(SHT3X_CMD['OTHER']['FETCH'], 6)
+            retries = 5
+            params = None
+            error = None
+            while retries > 0 and params is None:
+                try:
+                    params = self.i2c.i2c_read(
+                        SHT3X_CMD['OTHER']['FETCH'], 6, retry=False
+                    )
+                except Exception as e:
```

Comment:
I think this should catch `self.i2c.get_mcu().error`, as catching any error could lead to inadvertently not propagating a more serious error.

Reply:
I just didn't find that attribute, thanks